code/modules/mob/living/simple_animal/bees/bees_mob.dm

```diff
@@ -31,9 +31,18 @@
 
 /obj/effect/decal/cleanable/bee/New()
 	..()
-	dir = pick(cardinal)
-	pixel_x = rand(-10,10)
-	pixel_y = rand(-4,4)
+	var/image/I = image(icon,icon_state)
+	I.pixel_x = rand(-10,10)
+	I.pixel_y = rand(-4,4)
+	I.dir = pick(cardinal)
+
+	for (var/obj/effect/decal/cleanable/bee/corpse in get_turf(src))
+		if (corpse != src)
+			corpse.overlays += I
+			qdel(src)
```

Comment:
this will call `qdel(src)` multiple times if you have 3 or more in one turf

Reply:
yeah, use a break after this